fdbclient/ClientLogEvents.h

```diff
@@ -45,6 +98,7 @@ namespace FdbClientLogEvents {
 		double startTs{ 0 };
 
 		void logEvent(std::string id) const {}
```

Comment:
Both logEvent and addToRequestStats can be made as pure virtual function.

Reply:
I made them both virtual, but I think it makes sense to leave a base implementation that does nothing, since some events may not have any effect on request stats or tracing.